nixops/state/json_file.py

```diff
@@ -0,0 +1,356 @@
+# -*- coding: utf-8 -*-
+
+import nixops.deployment
+from nixops.state.state_helper import _create_resource_state
+import os
+import os.path
+import sys
+import threading
+import fcntl
+import re
+import json
+import copy
+
+from uuid import uuid1 as gen_uuid
+
+class TransactionalJsonFile:
```

Comment:
There should be a motivation for the existence of this class. It shouldn't just say what it does (it does that quite well), but why it exists in this first place.

Reply:
@coretemp @moretea how would you change the comment?